src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -1494,4 +1494,7 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Project_of_ID_0_is_required_to_accomplish_the_task_but_is_not_available_from_the_solution" xml:space="preserve">
     <value>Project of ID {0} is required to accomplish the task but is not available from the solution</value>
   </data>
+  <data name="_0_is_in_a_different_project" xml:space="preserve">
```

Comment:
We are past loc deadline, @tmeschter is this OK if this still goes in? This is for a message that would be for an exception being thrown from our API, so won't be seen in Visual Studio unless something is going very wrong.

Reply:
This is going into 16.6 so question is moot.